.github/workflows/reusable-solidity-docs.yml

```
@@ -0,0 +1,234 @@
+        if: inputs.exportAsGHArtifacts == true
+        uses: actions/upload-artifact@v4
+        with:
+          name: contracts-final-output
+          path: .${{ inputs.projectDir }}/generated-docs
+
+      - name: Post link to artifacts in PR comment
+        if: |
+          inputs.exportAsGHArtifacts == true
+            && inputs.commentPR == true
+            && startsWith(github.ref, 'refs/pull')
+        uses: actions/github-script@v6
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)